share/wake/lib/system/remote_cache_api.wake

```diff
@@ -673,12 +767,30 @@ def getPathAsJson path =
         "hash" :-> JString path.getPathHash,
     )
 
+# Helper function for further processing the result of a http request in relation to the rsc
+def matchRequestResult (label: String) (attempt: Result HttpResponse Error): Result HttpResponse Error =
+    match attempt
+        Pass r -> Pass r
+        Fail err ->
+            def cause = err.getErrorCause
+
+            if cause ==* "curl timed out (exit code 28)" then
```

Comment:
I've said before that I don't like matching against the Stringy cause, but given the fact that we'd need to open a hole in `package http` in order to get anything else, I think this is good enough here.